### ISSUE TITLE
Clear Formatting doesn't clear paragraph styles

### ISSUE THREAD

**Issue by konrad147** [froala/wysiwyg-editor#3974]
Clear formatting action does not clear the `style` attribute from paragraphs. The original issue is there #3285
It was closed but the bug still exists.

**Comment by yellow1912** [comment]
Yup, I can confirm this one. It's definitely a bug that should be fixed.

**Comment by mtran-method** [comment]
May I ask if there's any update on this? Still having issues on our end.

**Comment by diegoag64** [comment]
Are there any updates on this? I am waiting for a fix also

**Comment by rcameroon** [comment]
We really need this fix, any updates would be appreciated...

**Comment by Chetan77verma** [comment]
It is a bug, Are there any updates on this ?

**Comment by ilyaskarim** [comment]
Fixed in https://froala.com/wysiwyg-editor/changelog/#4.0.4
